packages/apps/graph/src/graph/query/fungible-accounts-by-public-key.ts

```diff
@@ -0,0 +1,109 @@
+import { prismaClient } from '@db/prisma-client';
+import { getFungibleChainAccount } from '@services/account-service';
+import { chainIds } from '@utils/chains';
+import { normalizeError } from '@utils/errors';
+import { builder } from '../builder';
+import FungibleAccount from '../objects/fungible-account';
+import type { FungibleChainAccount } from '../types/graphql-types';
+import { FungibleAccountName } from '../types/graphql-types';
+
+builder.queryField('fungibleAccountsByPublicKey', (t) =>
+  t.field({
+    description: 'Retrieve an account by public key.',
+    args: {
+      publicKey: t.arg.string({
+        required: true,
+        validate: {
+          minLength: 1,
+        },
+      }),
+      fungibleName: t.arg.string({
+        required: false,
+        validate: {
+          minLength: 1,
+        },
+      }),
+    },
+    type: [FungibleAccount],
+    nullable: true,
+    async resolve(__parent, args) {
+      try {
+        const accountAndFungibleNames =
+          await getAccountAndFungibleNamesByPublicKey(
+            args.publicKey,
+            args.fungibleName ?? undefined,
```

Comment:
can't we do just this here since the function param already is defined as `fungible?`

```suggestion
            args.fungibleName,
```

Reply:
Its this weird case where the the argument can be `string | undefined | null`. I will make the param of the function `fungible?: string | null` since it might be cleaner
![image](https://github.com/kadena-community/kadena.js/assets/112877112/adf4f509-097e-46d0-8456-2a814ace10e2)